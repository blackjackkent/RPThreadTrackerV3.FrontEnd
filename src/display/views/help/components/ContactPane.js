import React from 'react';
import { TabPane, CardHeader, CardBody } from 'reactstrap';
import Card from '../../../shared/styled/Card';

class ContactPane extends React.Component {
	constructor() {
		super();
		this.state = {};
	}

	render() {
		return (
			<TabPane tabId="contact">
				<Card>
					<CardHeader>
						<i className="fas fa-envelope" /> Contact Me
					</CardHeader>
					<CardBody className="card-body">
						<p>
							Have a suggestion about the site? Encountered a bug? Want to just say hi
							or give me a hug? Click the button below to open an issue on GitHub with
							your message.
						</p>
						<div className="text-center">
							<a
								className="btn btn-primary"
								target="_blank"
								rel="noopener noreferrer"
								href="https://github.com/blackjackkent/RPThreadTrackerV3.FrontEnd/issues/new"
							>
								Go to Github
							</a>
						</div>
						<br />
						<p>
							You can also visit the tracker{' '}
							<a
								target="_blank"
								rel="noopener noreferrer"
								href="http://tblrthreadtracker.tumblr.com"
							>
								support blog&nbsp;
							</a>
							to see if your question has recently been answered.
						</p>
					</CardBody>
				</Card>
			</TabPane>
		);
	}
}

export default ContactPane;
