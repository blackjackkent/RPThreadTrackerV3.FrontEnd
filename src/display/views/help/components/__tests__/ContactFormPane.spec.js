// #region imports
import React from 'react';
import { shallow } from 'enzyme';
import ContactPane from '../ContactPane';
// #endregion imports

const createTestProps = (propOverrides) => ({
	...propOverrides
});

describe('rendering', () => {
	describe('snapshots', () => {
		it('should render valid snapshot', () => {
			const element = shallow(<ContactPane {...createTestProps()} />);
			expect(element).toMatchSnapshot();
		});
	});
});
