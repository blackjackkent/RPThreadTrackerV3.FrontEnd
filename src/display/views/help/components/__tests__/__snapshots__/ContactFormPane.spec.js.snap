// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rendering snapshots should render valid snapshot 1`] = `
<TabPane
  tabId="contact"
  tag="div"
>
  <Styled(Card)>
    <CardHeader
      tag="div"
    >
      <i
        className="fas fa-envelope"
      />
       Contact Me
    </CardHeader>
    <CardBody
      className="card-body"
      tag="div"
    >
      <p>
        Have a suggestion about the site? Encountered a bug? Want to just say hi or give me a hug? Click the button below to open an issue on GitHub with your message.
      </p>
      <div
        className="text-center"
      >
        <a
          className="btn btn-primary"
          href="https://github.com/blackjackkent/RPThreadTrackerV3.FrontEnd/issues/new"
          rel="noopener noreferrer"
          target="_blank"
        >
          Go to Github
        </a>
      </div>
      <br />
      <p>
        You can also visit the tracker
         
        <a
          href="http://tblrthreadtracker.tumblr.com"
          rel="noopener noreferrer"
          target="_blank"
        >
          support blogÂ 
        </a>
        to see if your question has recently been answered.
      </p>
    </CardBody>
  </Styled(Card)>
</TabPane>
`;
